cmd/rpc/main.go

```diff
@@ -33,9 +33,6 @@ func main() {
 	dir := filepath.Join(config.QlcTestDataDir(), uuid.New().String())
 	cm := config.NewCfgManager(dir)
 	cfg, err := cm.Load()
-	if cfg.RPC.Enable == false {
```

Comment:
should check `config.rpc.enable` before create rpc service.

Reply:
yes , it's a rpc test command and run in a temporary path everytime，and  default rpcEnabled is false